package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "broccoli-merge-trees": "^1.1.1",
-    "broccoli-persistent-filter": "1.2.0",
+    "broccoli-persistent-filter": "^1.0.0",
```

Comment:
could happen when installed with `npm@3` and some other plugin using `"broccoli-persistent-filter": "1.0.0"`